spec/models/user_spec.rb

```diff
@@ -416,20 +416,25 @@
   end
 
   describe "self.search" do
-    it "find users by email" do
-      user1 = create(:user, email: "larry@consul.dev")
-      create(:user, email: "bird@consul.dev")
-      search = User.search("larry@consul.dev")
+    describe "find users" do
+      let(:larry) { create(:user, email: "larry@consul.dev", username: "Larry Bird") }
```

Comment:
Note this test passes because the user is only created when the `larry` is accessed.

```ruby
      it "by email" do
        expect(User.search("larry@consul.dev")).to eq []
        expect(User.search("larry@consul.dev")).to eq [larry]
      end
```

Here's a fix:

```suggestion
      let!(:larry) { create(:user, email: "larry@consul.dev", username: "Larry Bird") }
```